pkgs/games/arx-libertatis/default.nix

```diff
@@ -51,10 +53,10 @@ stdenv.mkDerivation rec {
       first-person role-playing game / dungeon crawler
       developed by Arkane Studios.
     '';
-    homepage = http://arx-libertatis.org/;
-    license = licenses.gpl3;
+    homepage    = http://arx-libertatis.org/;
```

Comment:
please change this spacing back to just a single space between the attribute and `=` to match other packages.

Reply:
I have been doing this is almost all the packages I have written. It feels tidier.

Reply:
Actually the nixpkg [manual](https://nixos.org/nixpkgs/manual/#chap-conventions) says attribute names should be aligned.

Reply:
ping
If you insist I'll revert this change, the package should be updated to fix the Qt runtime errors.

Reply:
I reverted the changes since I received no reply. Hope we can merge this and backport it to 19.09 soon.